src/main/java/com/mercadolibre/grupo1/projetointegrador/entities/PurchaseOrder.java

```diff
@@ -1,4 +1,39 @@
 package com.mercadolibre.grupo1.projetointegrador.entities;
 
+import com.mercadolibre.grupo1.projetointegrador.entities.enums.OrderStatus;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * classe responsavel por registrar os informacoes de uma compra do cliente, sendo utilizada
+ * desde o momento que o cliente abre o carrinho de compra (com StatusOrder de "OPENED") até
+ * o fechamento da depois que o produto é entregue
+ * @Author: Rogerio Lambert
+ */
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public class PurchaseOrder {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(mappedBy = "purchaseOrder", cascade = CascadeType.ALL)
+    private List<PurchaseItem> products;
+    private LocalDateTime createdDate;
+    private  LocalDateTime updatedDate;
+    @ManyToOne
+    private Customer customer;
+    private OrderStatus orderStatus;
```

Comment:
Bota o enum como string se nn ele vai pegar o valor ordinal, tipo: 0,1,2...
```suggestion
    @Enumerated(EnumType.STRING)
    private OrderStatus orderStatus;
```

Reply:
Concordo com a sugestão , já coloquei  a anotação @Enumerated do enum